motions:
  unfold:
    joint_names:
      - "girona1000/bravo/joint1"
      - "girona1000/bravo/joint2"
      - "girona1000/bravo/joint3"
      - "girona1000/bravo/joint4"
      - "girona1000/bravo/joint5"
      - "girona1000/bravo/joint6"
    points:
      - positions: [-2.72, -0.05, -1.51, 0.0, -3.1, 1.9]
        velocities: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
        time_from_start: {secs: 1.0, nsecs: 0.0}
      - positions: [-2.12, 0.370, -0.97, -1.41, -2.61, 2.035]
        velocities: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
        time_from_start: {secs: 3.0, nsecs: 0.0}
      - positions: [-1.92, 0.03, -0.28, -1.98, -1.69, 0.0]
        velocities: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
        time_from_start: {secs: 6.0, nsecs: 0.0}

  fold:
    joint_names:
      - "girona1000/bravo/joint1"
      - "girona1000/bravo/joint2"
      - "girona1000/bravo/joint3"
      - "girona1000/bravo/joint4"
      - "girona1000/bravo/joint5"
      - "girona1000/bravo/joint6"
    points:
      - positions: [-1.92, 0.03, -0.28, -1.98, -1.69, 1.57]
        velocities: [0.0,  0.0, 0.0, 0.0, 0.0, 0.0]
        time_from_start: {secs: 4.0, nsecs: 0.0}
      - positions: [-2.12, 0.370, -0.97, -1.41, -2.61, 2.035]
        velocities: [-0.1, -0.1, -0.1, 0.1, -0.1, 0.1]
        time_from_start: {secs: 8.0, nsecs: 0.0}
      - positions: [-2.72, -0.05, -1.51, 0.0, -3.1, 1.9]
        velocities: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
        time_from_start: {secs: 13.0, nsecs: 0.0}
